include/deal.II/fe/fe_values.h

```diff
@@ -32,6 +32,7 @@
 
 #include <deal.II/fe/fe.h>
 #include <deal.II/fe/fe_update_flags.h>
+#include <deal.II/fe/fe_values_base.h>
 #include <deal.II/fe/fe_values_extractors.h>
 #include <deal.II/fe/mapping.h>
 #include <deal.II/fe/mapping_related_data.h>
```

Comment:
Shouldn't we add here also fe_values_views.h to keep backward compatibility with user codes?

Reply:
Maybe? What we have now is backwards-compatible because that header file is included by `fe_values_base.h`. We could add it to be explicit.

Since setting up `FEValuesViews` objects is the responsibility of the base class I think it should remain a transitive inclusion.